src/styles.scss

```diff
@@ -79,3 +79,7 @@ ion-item-divider {
 ion-label {
   white-space: normal !important;
 }
+
+img {
+  border-radius: 100%;
```

Comment:
Just a thought for the future. We should really move away from global styles, especially on tags. Common styles should be moved into classes with some global prefix so we know where those styles are coming from.